quicfire_tools/inputs.py

```diff
@@ -117,6 +118,27 @@ def from_dict(cls, data: dict):
         return cls(**data)
 
 
+class OutputTimes(BaseModel):
```

Comment:
Do we need a class for this? I think it would be better to include these as individual attributes to QUIC_Fire. We can have a single output_time argument to SimulationInputs which writes all of these for the base case.

Reply:
I had the class in there so that if the user wanted the same value for all 4 output times (common in my experience) they wouldn't have to put it in 4 different arguments. But if the SimulationInputs class would handle this then you're right it is redundant and we should just have individual attributes